docs/README.md

```diff
@@ -1,11 +1,29 @@
-# Tablesaw Documentation
+## Compiled Docs
```

Comment:
docs/README.md is what's rendered at https://jtablesaw.github.io/tablesaw/  Perhaps it'd be possible to reconfigure it, but I'm not sure. Maybe it'd be better to add this to docs/contributing.md?

Reply:
Nevermind. This was the first file I looked at so I hadn't noticed that you were moving everything

Reply:
Since the docs are now being built what we would actually deploy is in the `docs/dist/` directory. We would need to change the root directory for the docs to from `docs/` to `docs/dist`

The reason you see a diff here is because I moved all our old markdown files to `docs/main/`  and used the docs/README to explain how to build the docs.